### DISCUSSION TITLE
Long PS / Group 15A

### DISCUSSION THREAD

**Discussion by lxsxl** [PE5PVB/TEF6686_ESP32#309] [Ideas]
Did you think about implementing the long PS function? Yet I have not seen any station using this feature, so for me it is not urgent and not important for the rc of 2.00. But maybe a thing for the future developements...

**Comment by PE5PVB** [comment]
I've never heard of a station using that. I thought this was moved to RDS2. If you have a MPX recording with long PS in RDS standard, I can add it.

**Reply by lxsxl** [reply]
I don't have one. So skip it for now. We will see if something is coming up.

**Comment by PE5PVB** [comment]
Do you know a software-basic RDS generator which can do this and outputs MPX over soundcard?

**Comment by lxsxl** [comment]
I think you are right that the long PS function has come with RDS2-definition. But as it uses group type 15A it would be useable with standard RDS receivers (that don't receive the three higher subcarriers). Actually I have never tried any encoder software. Searchengine reveals this one: https://github.com/Anthony96922/MiniRDS, idk if it could help?

**Reply by PE5PVB** [reply]
This software is to control a harware encoder. I need something which can generate RDS via souncard.

**Comment by lxsxl** [comment]
Okay, I understand. Next try: This one https://trx011.com/download/bestanden/TRX011_Setup_V2xx.exe is at least usable for long PS / group 15A. I wonder if it will help, cause maybe you also need the MacBe hardware. But there is a demo mode, maybe you are successful...

Btw, as the RC soon will be reality: What about expanding EON information? Is this a developement at all or rather for a later version or impossible as a further screen would be needed?

**Comment by lxsxl** [comment]
I also tried Magic RDS (from pira.cz) but long PS / group 15A is not supported. So if the one above doesn't work, I don't have any further idea.

**Comment by PE5PVB** [comment]
OK. Let's close this for now, since I don't know any stations who transmits it, and I can't test it. If you have ideas in the future please open it again.
